CONTRIBUTING.md

```diff
@@ -12,3 +12,19 @@
 #### EditorConfig
 
 We use [EditorConfig](http://EditorConfig.org) to maintain consistent coding styles between various editors and IDEs.  You can find our settings in the `.editorconfig` file in the root of the project.
+
+## PR review
+
+PRs can be be merged by any member of the [Podio Community](https://github.com/orgs/podio-community) organization, when the following conditions are met:
+
+1. All build checks are ok/green.
+2. At least 2 members (different to the PR creator) reviewed the PR.
+3. All questions/problems are addressed.
+
+## Versioning
+
+Versioning generally follows: https://semver.org/. New versions should be cut as often as neccessary. With each release `CHANGES.md` is updated.
+
+## Feature Requests / Issues
+
+New feature requests can be posted as [issues](https://github.com/podio-community/podio-php/issues) (please search for similar issues first!). Generally anyone can work on any issue. If you actively work on an issue, please leave a comment there, so there'll be less duplicate work. Issues are resolve on a best effort basis. As this project only consists of vuluntary contributors, there won't be any time lines (except if you commit yourself to one ;-) ).
```

Comment:
```suggestion
New feature requests can be posted as an [issue](https://github.com/podio-community/podio-php/issues) (please search for similar issues first!). Generally, anyone can work on any issue. If you actively work on an issue, please leave a comment on the issue, to avoid duplicate work. Issues are resolved on a best effort basis. As this project only consists of voluntary contributors, there won't be any timelines (except if you commit yourself to one ;-)).
```